searchcore/src/vespa/searchcore/proton/reprocessing/reprocess_documents_task.cpp

```diff
@@ -30,55 +28,40 @@ ReprocessDocumentsTask(IReprocessingInitializer &initializer,
     }
 }
 
-
 void
 ReprocessDocumentsTask::run()
 {
     if (_handler.hasProcessors()) {
-        EventLogger::reprocessDocumentsStart(_subDbName,
-                                             _visitorCost);
-        fastos::TimeStamp ts(fastos::ClockSystem::now());
-        _startTime = ts.ms();
-        _loggedTime = _startTime;
+        EventLogger::reprocessDocumentsStart(_subDbName,_visitorCost);
+        _stopWatch.start();
         search::IDocumentStore &docstore = _sm->getBackingStore();
         if (_handler.hasRewriters()) {
-            docstore.accept(_handler.getRewriteVisitor(),
-                    *this,
-                    *_docTypeRepo);
+            docstore.accept(_handler.getRewriteVisitor(),*this,*_docTypeRepo);
         } else {
-            docstore.accept(_handler,
-                    *this,
-                    *_docTypeRepo);
+            docstore.accept(_handler,*this,*_docTypeRepo);
```

Comment:
Some missing comma whitespace in this function

Reply:
Fixed